docs/autocomplete-definition.rst

```diff
@@ -175,3 +175,26 @@ In a multi-site context, you *have* to define a preemptive filter to gather info
 
     You may have noticed that these two classes have a ``model`` property and a ``requires_authentication`` property set to ``True``. Because we're using a user-based context, the ``requires_authentication`` will allow the general "out-of-context code" (form class creation) to instanciate the Agnocomplete class without the context, but will disallow it to return results based on the ``query``. This way, you can filter out unauthorized uses of the autocomplete, as you could do it with the ``@login_required`` decorator in
     your views.
+
+
+Custom the label
+----------------
+
+Sometimes, it can be useful to custom the display label. The class provide a
+method that can be override:
+
+.. code-block:: python
+
+    class AutocompletePerson(AgnocompleteModel):
+    model = Person
```

Comment:
I think that your indent is wrong. Please check using the ``tox -e docs`` job and browse the generated documentation to make sure everything is ok.